### PR TITLE
docs: update packages/theme docs

### PR DESCRIPTION
**Pull request by kyhy**
## PR checklist

- [x] I have followed the ["Committing and publishing"](https://github.com/quid/refraction/blob/master/CONTRIBUTING.md) guidelines;
- [x] I have added unit tests to cover my changes;
- [x] I updated the documentation and examples accordingly;

## Changes description
Updated import path in the packages/theme example

## Affected packages
- @quid/theme

### CODE REVIEW

File: packages/theme/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Once a styled component is wrapped by the ThemeProvider, it can access
 the current theme from the `props.theme` property.
 
 Additionally, inside the theme object, you will find a `current` property
-that will be set to `light` or `dark` depending by the active theme.
```
Comment (FezVrasta) [inline packages/theme/README.md]: Two trailing whitespaces are interpreted as newline in markdown, may you revert this change please?

Comment (kyhy) [reply packages/theme/README.md]: Hmm... is this whitespace different than space characters?

Comment (FezVrasta) [reply packages/theme/README.md]: No, they are just two white spaces. But markdown interprets 2 trailing white spaces this way.

Comment (kyhy) [reply packages/theme/README.md]: done
